.gitignore

```diff
@@ -105,3 +107,16 @@ ENV/
 
 /deeplabv3_pascal_trainval_2018_01_04.tar.gz
 /.pytest_cache/
+.DS_Store
+*.png
+*.jpg
+*.pbtxt
+*.ckpt*
+*checkpoint
+*.tfevents*
+training/max-image-segmenter-model-building-code.zip
```

Comment:
Duplicate. Is already covered in line 2 (which is automatically added by the starter kit installation program)